Utilities/Logging.cs

```diff
@@ -112,18 +114,38 @@ public static void incrementLogs()
             }
         }
 
-        private static void Report(ErrorLevel errorLevel, string message, object data = null, [CallerMemberName] string memberName = "", [CallerFilePath] string filePath = "")
+        private static async void Report(ErrorLevel errorLevel, string message, object originalData = null, [CallerMemberName] string memberName = "", [CallerFilePath] string filePath = "")
         {
             message = Redaction.RedactEnvironmentVariables(message);
-            Dictionary<string, object> thisData = PrepRollbarData(ref data);
-            if (thisData != null)
+            Dictionary<string, object> preppedData = PrepRollbarData(ref originalData);
+            if (originalData is null || preppedData != null)
```

Comment:
Sorry, I don't follow why we've added `originalData is null` here.

Reply:
As originally written, a message without a data package would resolve to false here and wouldn't be sent to Rollbar. This removes the requirement to include a data package with the message.